.travis.yml

```diff
@@ -1,10 +1,12 @@
 sudo: false
 language: go
 go:
-- 1.9.x
-- 1.10.x
 - 1.11.x
```

Comment:
Twirp promises to support the last two minor versions, just like Go's promise with security updates. We can't stop supporting go1.10.

Reply:
> Twirp promises to support the last two minor versions

Which patch versions of the minor versions?  modules are only in the latest patch (1.9.7 not 1.9.6 for example) of older versions